documentation/manual/src/main/asciidoc/en-US/modules/redis.asciidoc

```diff
@@ -36,6 +38,8 @@ The following properties are available to configure Redis support in Hibernate O
 .Redis datastore configuration properties
 hibernate.ogm.datastore.provider::
 To use Redis as a datastore provider, this property must be set to `redis_experimental`
+hibernate.ogm.datastore.grid_dialect::
+The dialect type how Hibernate OGM Redis maps entities and associations within Redis. The following two dialects exist: `org.hibernate.ogm.datastore.redis.RedisJsonDialect` (store entities as JSON documents within Redis keys. Associations are mapped to Redis lists/sets) and `org.hibernate.ogm.datastore.redis.RedisHashDialect` (store entities as key-value pairs within Redis hashes. Associations are mapped as JSON documents to Redis lists/sets)
```

Comment:
> Associations are mapped as JSON documents to Redis lists/sets

Is that true actually? Isn't this only the case for composite keys?

Btw. I recommend to just have one sentence per line. This keeps lines shorter and easier to diff down the road.